code/game/objects/items/devices/radio/beacon.dm

```diff
@@ -69,8 +69,8 @@
 		to_chat(user, "<span class='notice'>Locked In</span>")
 		new /obj/machinery/power/singularity_beacon/syndicate( user.loc )
 		playsound(src, 'sound/effects/pop.ogg', 100, 1, 1)
+		user.drop_item()
```

Comment:
might want to force this. Antidrop can probably fuck it up

Reply:
It'll still properly delete--it just might hard delete in this circumstance---sometimes it will, sometimes it work; for general circumstances like this, it'll be good enough.